src/applications/combined-debt-portal/medical-copays/containers/HTMLStatementPage.jsx

```diff
@@ -34,9 +34,7 @@ const HTMLStatementPage = ({ match }) => {
   const fullName = userFullName.middle
     ? `${userFullName.first} ${userFullName.middle} ${userFullName.last}`
     : `${userFullName.first} ${userFullName.last}`;
-  const acctNum = selectedCopay?.pHAccountNumber
-    ? selectedCopay?.pHAccountNumber.toString()
-    : selectedCopay?.pHCernerAccountNumber.toString();
+  const acctNum = selectedCopay?.accountNumber;
```

Comment:
I'd just double check on a potential fallback value, or include maybe a Sentry error to throw saying we're missing it? (without capturing any PII)

Reply:
@penny-lischer, I think what we want here is something like this:

`selectedCopay?.accountNumber || selectedCopay?.pHAccountNumber`

This will give us the fallback we need prioritizing the value we want and then using the value we had originally.